packs/ads/web/ads-web-gpt/config.yml

```diff
@@ -0,0 +1,19 @@
+name: ads-web-gpt
+title: Agent for Google Publisher Tags
+description: Agent to monitor web applications using GPT.
+summary: Agent to monitor web applications using GPT.
+level: Community
+authors:
+  - New Relic
```

Comment:
We have been using New Relic Labs for contributions from NR Labs. Can you update the author?

Reply:
Done!